languages/de.ini

```
@@ -452,6 +452,7 @@ Exception = "Exception"
 Excerpt = "Ausschnitt"
 exclude_facet = "[ausschließen]"
 exclude_newspapers = "Ohne Zeitungsartikel"
+excluded = "Ausgeschlossen:"
```

```suggestion
excluded = "Ausgeschlossen:"
```
The translator approved this translation.